Sources/Charcoal/Extensions/StringExtensions.swift

```diff
@@ -6,4 +6,29 @@ extension String {
     func localized(withComment comment: String = "") -> String {
         return NSLocalizedString(self, tableName: nil, bundle: .charcoal, value: "", comment: comment)
     }
+
+    func removingWhitespaces() -> String {
+        let components = self.components(separatedBy: .whitespaces)
+        return components.joined(separator: "")
+    }
+
+    mutating func removeWhitespaces() {
+        self = removingWhitespaces()
+    }
+
+    func range(from range: NSRange, replacementString: String) -> Range<String.Index>? {
```

Comment:
Can this be named to someway describe more what it does? Since the whole removing whitespace isn't exactly standard.

Reply:
Do you have any good names in mind? Perhaps `rangeConsideringWhitespaces` or something like that?

Reply:
Yes, maybe something like that, then it is obvious you use it for "special" things (avoid using this by mistake somewhere else). Or is it `replacementRangeNotConsideringWhitespaces` 🤔 are we considering the whitespaces or skipping them? 😂 

`rangeWithSpecialWhitespaceHandling`? 😛

Reply:
This func makes a range bigger so both the whitespace and the number before it will be removed. But I think `replacementRangeNotConsideringWhitespaces` should work.

Reply:
Renamed 😉